src/main/java/uk/gov/ons/census/casesvc/service/QidReceiptService.java

```diff
@@ -72,6 +73,15 @@ public void processUnreceipt(
       ResponseManagementEvent unreceiptEvent,
       OffsetDateTime messageTimestamp,
       UacQidLink uacQidLink) {
+    if (!uacQidLink.isActive() && hasEqReceipt(uacQidLink)) {
```

Comment:
do we need to check for an eq receipt that has come via any UacQid links not just this one?

Reply:
It says "Blank Questionnaire received and valid receipt received with same QID" on the ticket, but it's a possible scenario, sure.

Reply:
Blank questionnaires should already be logged and ignored if another QID has been receipted so I think this scenario only applies to receipts for the same QID?

Reply:
yeah that's a fair point